test/integration/scheduler/framework_test.go

```diff
@@ -944,18 +948,32 @@ func TestBindPlugin(t *testing.T) {
 				t.Errorf("test #%v: Didn't expected the postbind plugin to be called %d times.", i, postbindPlugin.numPostbindCalled)
 			}
 		}
+
+		expectInvokeEvents := map[pluginInvokeEvent]struct{}{}
```

Comment:
The invocation of bind and unreserve plugins must happen in a specific order. The fact that we see them happening in different orders could possibly point to a bug in the implementation.

Reply:
Let's see if it still happens after #80341 merged into master.